rpc/client/client_test.go

```diff
@@ -214,6 +215,77 @@ func TestGetBlock(t *testing.T) {
 	require.NoError(err)
 }
 
+func TestGetCommit(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+	mockApp, rpc := getRPC(t)
+	mockApp.On("BeginBlock", mock.Anything).Return(abci.ResponseBeginBlock{})
+	mockApp.On("Commit", mock.Anything).Return(abci.ResponseCommit{})
+
+	block := getRandomBlock(1, 5)
+	err := rpc.node.Start()
```

Comment:
Is this necessary?

Reply:
Well the test still works without starting the rpc. But I think it's still the correct flow.